Integration test for Kubernetes (#1399)
google/keytransparency: deploy/kubernetes/README.md

> ## Delete mysql DB

Reviewer: ```suggestion
## Delete deployment
```

> To delete all data written to the DB you can run:

Reviewer: ```suggestion
To delete the deployment, run:
```

```
kubectl delete -k deploy/kubernetes/overlays/gke
```

> Alternatively, you use the kubernetes HTTP user interface:

Reviewer: ```suggestion
You can use the kubernetes HTTP user interface to delete specific components, e.g. the MySQL database:
```

Reviewer: Does this still work? By default, GKE disables the Kubernetes HTTP UI these days.

Author: Ah, it probably does not.
